chapters/functions.tex

```diff
@@ -1700,15 +1701,17 @@ \section{Function Inlining and Event Generation}\label{function-inlining-and-eve
 
 If {\lstinline!Inline = false!}, the model developer proposes to not inline the function.
 
+The default for inlining is tool-specific.
```

Comment:
Would probably look nicer if combined with the previous sentence to form a two-sentence paragraph.